recipe/magento2.php

```diff
@@ -59,8 +59,8 @@
 set('magento_version', function () {
     // detect version
     $versionOutput = run('{{bin/php}} {{release_or_current_path}}/bin/magento --version');
-    preg_match('/(\d+\.?)+$/', $versionOutput, $matches);
-    return $matches[0] ?? "2.0";
+    preg_match("/(?:Magento\s|Magento\sCLI\s)*((?:[0-9]+\.?)+)/i", $versionOutput, $matches);
```

Comment:
What about skipping everything before actual number? It will be more reliable.

Reply:
True, I'll find a regx wizard on my team to help with that.

Reply:
Done, thanks to @johmanx10 :partying_face: 

I also updated the "testst" in the PR message body.